GPU/Workflow/src/GPUWorkflowSpec.cxx

```diff
@@ -251,13 +253,22 @@ DataProcessorSpec getGPURecoWorkflowSpec(gpuworkflow::CompletionPolicyData* poli
         config.configCalib.matLUT = o2::base::MatLayerCylSet::loadFromFile(confParam.matLUTFile.c_str(), "MatBud");
       }
 
+      // load from file
       if (confParam.dEdxFile.size()) {
         processAttributes->dEdxSplines.reset(new TPCdEdxCalibrationSplines(confParam.dEdxFile.c_str()));
       } else {
         processAttributes->dEdxSplines.reset(new TPCdEdxCalibrationSplines);
       }
       config.configCalib.dEdxSplines = processAttributes->dEdxSplines.get();
 
+      if (!confParam.dEdxCorrFile.empty()) {
+        LOGP(info, "Loading dEdx correction file: {}", confParam.dEdxCorrFile);
+        processAttributes->dEdxCorrection.reset(new o2::tpc::CalibdEdxCorrection(confParam.dEdxCorrFile));
+      } else {
+        processAttributes->dEdxCorrection.reset(new o2::tpc::CalibdEdxCorrection());
```

Comment:
We should check with David if we cannot just leave this as a `nullptr`

Reply:
why would you leave this as nullptr. From what I have seen above, you actually check for the presense of the object and you would fail the dEdx computation if it is not present. I think having a default object here that doesn't apply any correction makes sense.
(The alternative would be to check inside the fillCluster function for the presense of the object and use it only if not nullptr).